Analyse_Birds.qmd

```diff
@@ -436,11 +450,389 @@ for (i in 1:NSim) {
 boxplot(diagZ[2,], col='dodgerblue3',ylab='#Rate of nest fate well-classified')
 mean(diagZ[2,])
 ```
+
+
 Here we have **58%** of good predictions, which is somewhat below the common threshold of 70% for a model to be considered good. This suggests that the model may need improvement.
 
 This result is not surprising given our initial data. Because they are from different studies some variables barely contains some species or years, etc... This could explain why our model is not suitable. 
 
 
+# II. ANOVA
+
+
+Analysis of variance (ANOVA) is one of the most widely used statistical methods in various disciplines. It allows to model the relationship between a quantitative dependent variable $Y$ according to the different modalities of one or more qualitative explanatory variables $Xs$.
+
+We will perform an ANOVA to see the effect of qualitatives variables (state, habitat, species, year) on the quantitative Initiation_Date in the Bird_dataset.csv dataset.
+
+We need to check the type of the variables :
+  
+```{r Ini_factor data}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Dataset : variables as factors
+
+data$State<-as.factor(data$State)
+data$Habitat<-as.factor(data$Habitat)
+data$Species<-as.factor(data$Species)
+data$Year<-as.factor(data$Year)
+str(data)
+
+```
+
+```{r Ini_missing_data}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Checking for missing data
+
+colSums(is.na(data))  
+```
+  
+There is no missing data here.
+  
+  
+Before any modeling, we must explore the data in order to avoid possible errors. 
+Below we present the list of steps to follow before proceeding to the modeling:
+
+- 1: Check for the presence of outliers in $Y$ and the distribution of the 
+values of $Y$
+
+- 2: If $X$ is a **quantitative** explanatory variable, check for the presence 
```

Comment:
I think these two lines can be removed because there is no quantitative variable

Reply:
Here the goal is to explain the conditions in a general way. The variables can be either quantitative or qualitative. Later I explain that all the variables Xs are qualitative. I find it important to expose the different possibilities to the reader.